### PR TITLE
New core consideration (plus continuation of the "Beginner Tutorials" section)

### PR DESCRIPTION
**Pull request by ritzip**
Will this work? I will work on others soon

### CODE REVIEW

**Comment by Luxray5474** [comment]
This folder is practically 'stale', but once @Anuken and I agree that this chould continue and more people come in to contribute I will merge it.

**Comment by Arkanic** [comment]
@Luxray5474 will there be a "mass update" at some point? (old content gets replaced with the newer stuff?)

**Comment by Anuken** [comment]
A beginner tutorial would be a good idea, but most of this page is outdated and needs a complete rewrite. It's also worth noting that I will be changing resource collection controls soon; I don't think it's worth updating until then.

File: docs/Beginner Tutorials/Basics.md [hunk 1]
```diff
@@ -90,4 +90,4 @@ While building, drones will stop assisting and quickly mine ores that are runnin
 - You can confirm a block, cancel the build (this will stop your mech from building, but the confirmed build will remain) and continue this one block a time to set up a large queue. This can be helpful when you want to get something started, but need to address something else first. Drones will not work on these queues, so this is only a placeholder.
 - Containers making contact with your Core will act like extra inventory for your Core. This not only gives more inventory space, but allows for more space to input raw materials, as well as output raw materials from your inventory (See Unloader) An Unloader will take materials out of both the Core or connected Containers because they now share the same inventory space.
 
-<img src="https://i.imgur.com/vcXy8EZ.png" width=40%>
\ No newline at end of file
+<img src="https://i.imgur.com/vcXy8EZ.png" width=40%>
```
Comment (Anuken) [inline docs/Beginner Tutorials/Basics.md]: This was already part of the old page, but I don't think it's a good idea to ever have imgur links here-- they can break after a while, as was demonstrated with the Mindustry README. Instead, images should be placed on the wiki itself and linked directly.

File: docs/Beginner Tutorials/Basics.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 **The goal of the game is to not die! You die when your core reaches 0 health.** This is your core:
 
-<img src="https://i.imgur.com/HOiVEDe.png" width=20% height=20%>
+<img src="https://raw.githubusercontent.com/Anuken/Mindustry/master/core/assets-raw/sprites/blocks/storage/core-shard.png" width=20% height=20%>
```
[the remark this answers is not in the document]
Comment (Arkanic) [reply docs/Beginner Tutorials/Basics.md]: @Anuken are all of the assets imported in from Mindustry or only certain ones?

[the remark this answers is not in the document]
Comment (Anuken) [reply docs/Beginner Tutorials/Basics.md]: Every spritesheet image is.

**Comment by ritzip** [comment]
Is it fine to just rewrite the outdated ones?
(And trying to improve it)

File: docs/Beginner Tutorials/Basics.md (hunk 1 above)
Comment (Luxray5474) [reply docs/Beginner Tutorials/Basics.md]: True!

**Comment by Luxray5474** [comment]
Of course. I would suggest a complete rewrite even. But again, after Anuke changes 

> resource collection controls soon

Then we could merge to a Beginner Tutorials branch here (not the old one) and make changes through there.

File: docs/Beginner Tutorials/Basics.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Arkanic) [reply docs/Beginner Tutorials/Basics.md]: @Anuken owoof

**Comment by Arkanic** [comment]
@Luxray5474 ok
I'll try to help with that
:D

**Comment by Arkanic** [comment]
When Mindustry 5.0 or whatever the final "in-dev" build is, will the main mindustrygame.github.io be edited to the most recent version?
if so, i'd like to help
~~I like HTML~~

**Comment by ritzip** [comment]
~~this is markdown~~

**Comment by Arkanic** [comment]
@ritzip 
~~the physical outdated page uses HTML~~
~~and i know about teh markdown~~

**Pull request closed without merging**